### PR TITLE
fix bug 1189596 - only remap urls under zones

### PR DESCRIPTION
**Pull request by groovecoder**

### CODE REVIEW

File: kuma/wiki/middleware.py [hunk 1]
```diff
@@ -31,7 +31,8 @@ def process_request(self, request):
         remaps = DocumentZoneURLRemapsJob().get(request.locale)
         for original_path, new_path in remaps:
 
-            if request.path_info.startswith(original_path):
+            if (request.path_info == original_path or
+                request.path_info.startswith(''.join([original_path, '/']))):
```
Comment (jezdez) [inline kuma/wiki/middleware.py]: the original path could be a Unicode string, so this should be `u''.join([original_path, '/'])` or just `u'%s/' % original_path`

**Comment by groovecoder** [comment]
Good catch. Updated code & test.

**Comment by groovecoder** [comment]
Rebased on master to see if latest Travis fix works well. :smile:

**Comment by jwhitlock** [comment]
+r

Tried:
```shell
curl -kI https://developer.mozilla.org/en-US/docs/Mozilla/Firefox_for_IOS # 302 Found
curl -kI https://developer-local.allizom.org/en-US/docs/Mozilla/Firefox_for_iOS # 200 OK
curl -kI https://developer-local.allizom.org/en-US/docs/Mozilla/Firefox # 302 Found
curl -kI https://developer-local.allizom.org/en-US/docs/Mozilla/Firefox/ # 302 Found
curl -kI https://developer-local.allizom.org/en-US/docs/Mozilla/Firefox_foo # 404 Not Found
```

**Pull request merged**